plugins/graphiql/src/lib/api/GraphQLEndpoints.ts

```diff
@@ -31,6 +31,10 @@ export type EndpointConfig = {
   method?: 'POST';
   // Defaults to setting Content-Type to application/json
   headers?: { [name in string]: string };
+  /**
+   * Fetch API to use instead of browser fetch()
+   */
+  fetchApi?: FetchApi;
```

Comment:
Nice. Can you also make sure at the same time that it does get automatically injected here too?

https://github.com/backstage/backstage/blob/bb8b11fef169ed32ea12bb4a350fa3980b36732c/packages/app/src/apis.ts#L58

As a nice extra, it'd be great to have it injected in the new backend system stuff too (in alpha.ts) ... but that takes a little bit of a refactor. So probably best to leave that for some other time.

Reply:
Yes, makes sense to use it there in the example app! That's pushed.

I had never seen that alpha stuff, looks interesting, but leaving out for now yeah.